lib/login/view/login_page.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:auth_example/login/bloc/login_bloc.dart';
+import 'package:auth_example/login/view/login_view.dart';
+import 'package:auth_service/auth.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LoginPage extends StatelessWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => LoginBloc(
```

Comment:
should we use repository provider for this?

Reply:
Repository providers should only be used if we are injecting data repositories to the bloc. this is what the documentation says. however in this case, we are purely providing bloc to widget tree so bloc provider.

Reply:
Yes, I meant the authService maybe we can put it as a RepositoryProvider<AuthService> and then inject that one into the bloc

Reply:
Also it would be for the signup